CMT_track_params :
    emd_threshold : 0.8
    min_depth : 0.1
    max_depth : 10.0
    
    
